ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -511,7 +511,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn-clusternetwork-cidr-expansion,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
how did this happen? moving **e2e-aws-ovn-edge-zones** to `always_run: false` ?

Reply:
Sorry I opened #66904 in liu of it. I figured I made some mistakes :(